app/assets/stylesheets/articles.scss

```diff
@@ -4,297 +4,74 @@
-            background,
-            theme-container-background-hover,
-            lighten($bold-blue, 38%)
-          );
-        }
-        &.selected {
-          @include themeable(
-            background,
-            theme-container-accent-background,
-            lighten($bold-blue, 8%)
-          );
-          @include themeable(color, theme-color, white);
-        }
-      }
+      max-width: calc(100% - 280px);
+      padding: 16px;
     }
-    .on-page-nav-controls {
-      height: 45px;
-      position: relative;
-      opacity: 0.85;
-      &.user-nav-controls {
-        margin-top: -12px;
-        @media screen and (min-width: 950px) {
-          display: none;
-        }
-      }
-      .on-page-nav-butt {
-        position: absolute;
-        top: 14px;
-        border: 0px;
-        background: transparent;
-        border-radius: 8px;
-        img {
-          height: 24px;
-        }
-        &.on-page-nav-butt-left {
-          left: 3%;
-        }
-        &.on-page-nav-butt-right {
-          right: 3%;
-        }
-        @media screen and (min-width: 950px) {
-          display: none;
-        }
-      }
-      .on-page-nav-label {
-        position: absolute;
-        right: 0;
-        left: 0;
-        font-family: $helvetica-condensed;
-        font-stretch: condensed;
-        top: 18px;
-        font-weight: 600;
-        &.sub-home-nav {
-          top: 12px;
-        }
-        .wide-nav-links {
-          display: none;
-        }
-        @media screen and (min-width: 650px) {
-          .narrow-nav-select {
-            display: none;
-          }
-          .wide-nav-links {
-            display: block;
-            cursor: default;
-          }
-        }
-        @media screen and (min-width: 950px) {
-          top: 9px;
-          &.sub-home-nav {
-            top: 8px;
-          }
-        }
-      }
-      .separator {
-        border-left: 2px solid $light-medium-gray;
-        display: inline-block;
-        margin: 0px 7px;
-        height: 15px;
-      }
-      .nav-chronofiter-link {
-        display: inline-block;
-        padding: 2px 8px;
-        font-size: 0.95em;
-        border-radius: 2px;
-        border: 1px solid transparent;
-        &.selected {
-          background: $purple;
-          color: $black;
-          border: 1px solid $dark-purple;
-          box-shadow: 3px 4px 0px $dark-purple;
-        }
-        &:hover {
-          opacity: 1;
-          background: $dark-purple;
-          color: white;
-          border: 1px solid $purple;
-          box-shadow: 3px 4px 0px $purple;
-        }
-      }
-      .narrow-nav-select {
-        button {
-          all: unset;
-          background: transparent;
-          border: 0;
-          font-family: $helvetica-condensed;
-          font-stretch: condensed;
-          font-weight: 600;
-          cursor: pointer;
-        }
-      }
+
+    @media screen and (min-width: 1120px) {
+      max-width: calc(100% - 280px - 280px);
```

Comment:
```suggestion
      max-width: calc(100% - 560px);
```

Reply:
I've written it that way to make it explicit it's related to two sidebar width. `560px` feels more random without any context. But I guess I can just add a comment there :)